pkg/docker-engine/Dockerfile

```diff
@@ -173,9 +172,9 @@ RUN --mount=type=bind,source=scripts/pkg-rpm-build.sh,target=/usr/local/bin/pkg-
     --mount=type=bind,source=scripts/check-gomod.sh,target=/usr/local/bin/check-gomod \
     --mount=type=bind,from=scripts,source=gen-ver.sh,target=/usr/local/bin/gen-ver \
     --mount=type=bind,from=scripts,source=fix-cc.sh,target=/usr/local/bin/fix-cc \
-    --mount=type=bind,source=common/systemd/docker.service,target=/root/rpmbuild/SOURCES/docker.service \
-    --mount=type=bind,source=common/systemd/docker.socket,target=/root/rpmbuild/SOURCES/docker.socket \
     --mount=type=bind,from=src-tgz,source=/out/engine.tgz,target=/root/rpmbuild/SOURCES/engine.tgz \
+    --mount=type=bind,from=src,source=/src/contrib/init/systemd/docker.service,target=/root/rpmbuild/SOURCES/docker.service \
+    --mount=type=bind,from=src,source=/src/contrib/init/systemd/docker.socket,target=/root/rpmbuild/SOURCES/docker.socket \
```

Comment:
Wondering if it's already extracting `engine.tgz` so if the files are already present ... somewhere and .. maybe not needed to be mounted separately, but if this works, we can look at that in a follow-up I guess 😅